arviz/plots/hdiplot.py

```diff
@@ -34,8 +36,11 @@ def plot_hdi(
     ----------
     x : array-like
         Values to plot.
-    y : array-like
+    y : array-like, optional
         Values from which to compute the hdi. Assumed shape (chain, draw, \*shape).
+        Only optional if hdi_data is present.
+    hdi_data : array_like, optional
+        Hdi values to use.
```

Comment:
I think the description should be a bit more explicit, especially w.r.t the difference with `y`. As such, I fear this could be mistaken for `hdi_prob`

Reply:
definitely, I have in mind to extend docs, also add a `See also` section pointing to `hdi` and maybe one example.